cli/default-reporter/test/reportingLifecycleScripts.ts

```diff
@@ -233,29 +233,50 @@ test('groups lifecycle output when append-only is used', (done) => {
 
   expect.assertions(1)
 
-  const allOutputs = [] as string[]
+  // const allOutputs = [] as string[]
 
-  output$.pipe(take(11), map(normalizeNewline)).subscribe({
-    complete: () => {
-      expect(allOutputs.join(EOL)).toBe(`\
-${chalk.cyan('packages/foo')} ${PREINSTALL}$ node foo
-${chalk.cyan('packages/foo')} ${PREINSTALL}: foo 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30
-${chalk.cyan('packages/foo')} ${PREINSTALL}: Failed
-${chalk.cyan('packages/foo')} ${POSTINSTALL}$ node foo
-${chalk.cyan('packages/foo')} ${POSTINSTALL}: foo I
-${chalk.magenta('packages/bar')} ${POSTINSTALL}$ node bar
-${chalk.magenta('packages/bar')} ${POSTINSTALL}: bar I
-${chalk.cyan('packages/foo')} ${POSTINSTALL}: foo II
-${chalk.cyan('packages/foo')} ${POSTINSTALL}: foo III
-${chalk.blue('packages/qar')} ${INSTALL}$ node qar
-${chalk.blue('packages/qar')} ${INSTALL}: Done`)
-      done()
-    },
-    error: done,
-    next: (output: string) => {
-      allOutputs.push(output)
-    },
-  })
```

Comment:
This test would never complete, because if the `expect(allOutputs.join(EOL))` fails, the succeeding `done()` would not be called because `expect` throws